docs/execution-providers/DirectML-ExecutionProvider.md

```diff
@@ -23,34 +23,70 @@ The DirectML Execution Provider currently uses DirectML version [1.8.0](https://
 * TOC placeholder
 {:toc}
 
+
+## Windows OS integration
+
+ONNX Runtime is available in Windows 10 versions >= 1809 and all versions of Windows 11. It is embedded inside Windows.AI.MachineLearning.dll and exposed via the WinRT API (WinML for short). It includes the CPU execution provider and the [DirectML execution provider](../execution-providers/DirectML-ExecutionProvider) for GPU support.
+
+The high level design looks like this:
+
+![ONNX + WinML layered architecture](../../images/layered-architecture.png)
+
+### API choice
+{: .no_toc }
+
+You can choose to use either the WinRT API or the C API.
+
+||WinRT|C API|
+|--|--|--|
+|Type system| Integration with Windows RT types| Platform neutral types|
+|Language support| Language support via WinRT Projections| Language support via per language projections|
+|Tensorization| Accepts VideoFrames and converts to tensors (support for CPU and GPU)| Accepts tensors|
+
+### Using the NuGet WinRT API with other C-API distributions
+{: .no_toc }
+
+The WinRT API NuGet package is distributed with a specific version of ONNX Runtime, but apps can include their own version of ONNX Runtime (either a [released version](../install/#cccwinml-installs) or [a custom build](../build/)). You may wish to do this to use non-default execution providers.
+To use your own version of ONNX Runtime, replace onnxruntime.dll with your desired version.
+
+<p><a href="#">Back to top</a></p>
+
+
 ## Install
-Pre-built packages of ORT with the DirectML EP is published on Nuget.org. See: [Install ORT](../install).
+Pre-built packages of ORT with the DirectML EP is published on Nuget.org. See: [Install ORT](../install#cccwinml-installs).
 
 ## Requirements
 
-The DirectML execution provider requires any DirectX 12 capable device. Almost all commercially-available graphics cards released in the last several years support DirectX 12. Examples of compatible hardware include:
+The [DirectML](https://github.com/microsoft/DirectML) execution provider requires a [DirectX 12](https://answers.microsoft.com/en-us/windows/forum/all/direct-x-120-download/98d1137a-653d-40eb-9863-3b474665add7) capable device. Almost all commercially-available graphics cards released in the last several years support DirectX 12. Here are some examples of compatible hardware:
```

Comment:
No need for a link from DirectML, since it was already used in the first paragraph of this page and it has a link there.

The link on `DirectX 12` doesn't seem helpful to me. I'd remove it.